### ISSUE TITLE
configure ffmpeg failed, ran fine on second try

### ISSUE THREAD

**Issue by mark-dce** [acozine/sufia-centos#27]
running `vagrant provision` on single_user branch

Vagrantfile (excerpt)

```
  # Enable provisioning with Ansible
   config.vm.provision "ansible" do |ansible|
     #ansible.verbose = 'vvv'
     ansible.groups = {
       "vagrant" => ["default"],
       "all_groups:children" => ["group1"],
     }
     ansible.extra_vars = {
       deploy_user: "vagrant",
       server_name: "localhost",
       rails_env: "development",
       bundle_path: "~/.bundle"
     }
     ansible.playbook = "provisioning/vagrant-develop.yml"
   end
```

Got this error

```
TASK: [ffmpeg | configure ffmpeg] ********************************************* 
failed: [default] => {"changed": true, "cmd": "cd /opt/install/ffmpeg_sources/ffmpeg && ./configure --extra-libs=\"-ldl\" --enable-gpl --enable-nonfree --enable-libfdk_aac --enable-libmp3lame --enable-libopus --enable-libvorbis --enable-libvpx --enable-libx264", "delta": "0:00:04.612339", "end": "2015-11-13 17:56:09.910513", "rc": 1, "start": "2015-11-13 17:56:05.298174", "warnings": []}
stdout: ERROR: opus not found using pkg-config

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
Include the log file "config.log" produced by configure as this will help
solve the problem.

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/Users/mark/vagrant-develop.retry

default                    : ok=129  changed=117  unreachable=0    failed=1   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

added `ansible.start_at_task = "ffmpeg | configure ffmpeg"` and re-ran vagrant provision (no other changes)

```
MARKs-MB:wustl-dev mark$ vagrant provision
==> default: Running provisioner: ansible...

PLAY [provision basic vagrant box] ******************************************** 

GATHERING FACTS *************************************************************** 
ok: [default]

TASK: [ffmpeg | configure ffmpeg] ********************************************* 
changed: [default]

TASK: [ffmpeg | make ffmpeg] ************************************************** 
```

etc.

**Comment by mark-dce** [comment]
Update - build completed successfully after re-starting at that task.

**Comment by acozine** [comment]
Closed by 5553193d0bed5dd5a83b75987c33fce323164443
